### DISCUSSION TITLE
Custom Delay for Waxed Copper Bulbs Based on Oxidation State

### DISCUSSION THREAD

**Discussion by Tomate0613** [Tomate0613/revert-23w46a#2] [Ideas]
Exploring the prospect of introducing varied delays based on the oxidation state could be a promising enhancement for waxed copper bulbs. A suggestion is to implement a delay system with 0 ticks for the 'normal' copper bulb and an incremental one-tick delay for each subsequent oxidation state.

In the event that Mojang opts not to reverse these changes, it's crucial to consider making this enhancement an optional setting. This ensures that users have the flexibility to choose whether they want to incorporate these delays into their gameplay.

Should Mojang decide to revert these changes, a contingency plan could involve transforming this mod into a small mod focused on refining copper bulb mechanics (and even other redstone stuff).

Your input on this idea is highly valued, and I encourage an open discussion to gather diverse perspectives. Share your thoughts and insights to collectively refine and shape this proposed enhancement.

**Comment by cootshk** [comment]
You should add this as a gamerule or a command to toggle it on/off

**Reply by Tomate0613** [reply]
Definitely. Not sure if it should be a (global) config or a gamerule though.

**Reply by cootshk** [reply]
I would prefer a gamerule
but keep a global config for the default gamerule value
